assets/src/components/detours/diversionPage.tsx

```diff
@@ -123,7 +123,13 @@ export const DiversionPage = ({
               Detour is not editable from this screen.
             </Alert>
           )}
-          {routingError && <RoutingErrorAlert />}
+          {routingError?.type === "no_route" && (
+            <RoutingErrorAlert>
+              You can&apos;t route to this location. Please try a different
+              point.
+            </RoutingErrorAlert>
+          )}
+          {routingError?.type === "unknown" && <RoutingErrorAlert />}
```

Comment:
Nit: What if the logic for what message to display lived in the `<RoutingErrorAlert />` instead of the outer level? So then at the top level, it would just look like:

```tsx
{routingError && <RoutingErrorAlert error={routingError} />}
```

Reply:
I prefer to leave it as is for now, but we could address this in a follow up PR if desired.